packages/client/src/api/api.ts

```diff
@@ -26,7 +28,22 @@ export async function signupWithEmailPassword(email: string, password: string):
  * @param {string} password - The password of the user.
  * @return {Promise<AxiosResponse<any, any>>} - A promise that resolves to the Axios response.
  */
-export async function loginWithEmailPassword(email: string, password: string): Promise<AxiosResponse<any, any>> {
-  const apiUrl = `/auth/login`;
-  return;
+export async function loginWithEmailPassword(email: string, password: string): Promise<AxiosResponse<IAuthenticationResult>> {
+  const axiosClient = AxiosClient.getInstance();
+  const axios = axiosClient.axiosInstance;
+  try {
+    const response = await axios.post<IAuthenticationResult>("/auth/login", {
+      email,
+      password,
+    });
+
+    return response;
+  } catch (e) {
```

Comment:
Let it pass we will capture it in the hook

Reply:
It is a good pratice to handle error right where it's generated or esle it might create difficulties while debugging as you maynot sometime reach at the root of cause